app/models/allocation.rb

```diff
@@ -1,12 +1,26 @@
 # frozen_string_literal: true
 
 class Allocation < ApplicationRecord
-  belongs_to :pom_detail
-
   attr_accessor :responsibility
 
+  scope :active_allocations, lambda { |nomis_offender_ids|
```

Comment:
Should the active_allocations scope check active field? Are this and the following scope back to front?

Reply:
Yep you are right - fixed